doc/release-process.md

```diff
@@ -25,13 +25,16 @@ Before every major release:
 * Update [`src/chainparams.cpp`](/src/chainparams.cpp) m_assumed_blockchain_size and m_assumed_chain_state_size with the current size plus some overhead.
 * Update `src/chainparams.cpp` chainTxData with statistics about the transaction count and rate. Use the output of the RPC `getchaintxstats`, see
   [this pull request](https://github.com/bitcoin/bitcoin/pull/12270) for an example. Reviewers can verify the results by running `getchaintxstats <window_block_count> <window_last_block_hash>` with the `window_block_count` and `window_last_block_hash` from your output.
-* Update version of `contrib/gitian-descriptors/*.yml`: usually one'd want to do this on master after branching off the release - but be sure to at least do it before a new major release.
 * In `configure.ac` and `build_msvc/bitcoin_config.h` on _the master branch_:
   - update `CLIENT_VERSION_MINOR` version
 * In `configure.ac` and `build_msvc/bitcoin_config.h` on _a new release branch_ (see [this commit](https://github.com/bitcoin/bitcoin/commit/742f7dd972fca3dd4a33cfff90bf901b71a687e7)):
   - update `CLIENT_VERSION_MINOR` version
   - set `CLIENT_VERSION_REVISION` to `0`
   - set `CLIENT_VERSION_IS_RELEASE` to `true`
+* Before branching off the release, clear the release notes and move them to the wiki (see "Write the release notes" below).
+* After branching off the release:
```

Comment:
Instead of nested lists with `*`, could use some headings? I.e.

````
## Updates

### Before every release candidate
...

### Before every major release

#### Before branch-off

...

#### After branch off (on master)

#### After branch off (on the major release branch)

## Building

### First time/New builders

Reply:
Done.